financial-connections/src/main/java/com/stripe/android/financialconnections/features/linkaccountpicker/LinkAccountPickerViewModel.kt

```diff
@@ -270,7 +291,7 @@ internal class LinkAccountPickerViewModel @AssistedInject constructor(
                 // Nothing to log here
             }
         }
-
+        if (payload.acquireConsentOnPrimaryCtaClick) acceptConsent()
```

Comment:
Super nitty!

```suggestion
        if (payload.acquireConsentOnPrimaryCtaClick) {
            acceptConsent()
        }
```

Reply:
adding this on a PR with small changes coming today!